src/pages/workspaces/List.js

```diff
@@ -47,40 +47,6 @@ const workspaceSubmissionStatus = ({ workspaceSubmissionStats: { runningSubmissi
   )
 }
 
-const WorkspaceMenuContent = ({ namespace, name, onClone, onShare, onDelete, onLock }) => {
-  const { workspace } = useWorkspaceDetails({ namespace, name }, ['accessLevel', 'canShare', 'workspace.isLocked'])
-  const canRead = workspace && Utils.canRead(workspace.accessLevel)
-  const canShare = workspace?.canShare
-  const isOwner = workspace && Utils.isOwner(workspace.accessLevel)
-  const isLocked = workspace?.workspace.isLocked
-  return h(Fragment, [
-    h(MenuButton, {
-      disabled: !canRead,
-      tooltip: workspace && !canRead && 'You do not have access to the workspace Authorization Domain',
-      tooltipSide: 'left',
-      onClick: onClone
-    }, [makeMenuIcon('copy'), 'Clone']),
-    h(MenuButton, {
-      disabled: !canShare,
-      tooltip: workspace && !canShare && 'You have not been granted permission to share this workspace',
-      tooltipSide: 'left',
-      onClick: onShare
-    }, [makeMenuIcon('share'), 'Share']),
-    h(MenuButton, {
-      disabled: !isOwner,
-      tooltip: !isOwner && ['You have not been granted permission to ', isLocked ? 'unlock' : 'lock', ' this workspace'],
-      tooltipSide: 'left',
-      onClick: onLock
-    }, isLocked ? [makeMenuIcon('unlock'), 'Unlock'] : [makeMenuIcon('lock'), 'Lock']),
-    h(MenuButton, {
-      disabled: !isOwner,
```

Comment:
This was missing a check for `isLocked`, which was on the other version of the menu.